addons/danger/functions/fnc_tacticsContact.sqf

```diff
@@ -155,13 +166,24 @@ if (
 ) exitWith {
     // execute assault ~ forced
     {
-        [_x, _buildings] call EFUNC(main,doAssaultMemory);
+        //[_x, _buildings] call EFUNC(main,doAssaultMemory);
+        _x setUnitPos "MIDDLE";
+        _x forceSpeed 3;
+        //_x doMove selectRandom _buildings;
         _x setVariable [QEGVAR(main,currentTask), "Assault (contact)", EGVAR(main,debug_functions)];
 
         // forced movement
         _x setVariable [QGVAR(forceMove), true];
-        [{_this setVariable [QGVAR(forceMove), nil]}, _x, 5 + random 4] call CBA_fnc_waitAndExecute;
-    } foreach _units select {getSuppression _x < 0.5};
+        [
+            {
```

Comment:
Check if unit is dead and restore old unitPos

Reply:
Good call. Fixed in latest push.